src/components/ui/modal/MyModal.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classes from './MyModal.module.css'
+const MyModal = (props) => {
+    const modalClasses = [classes.myModal]
+    if (props.visible) {
+        modalClasses.push(classes.active)
+    }
+    return (
+        <div onClick={() => props.setVisible(false)} className={modalClasses.join(' ')}>
+            <div onClick={(e) => e.stopPropagation()} className={classes.myModalContent}>
+                {props.children}
```

Comment:
3. Now you can delete this if-statement and move the click handling to a separate function:
```suggestion
    const closeModal = () => {
        setVisible(false);
    };
    return (
        <div onClick={() => closeModal} className={modalClasses.join(' ')}>
            <div onClick={(e) => e.stopPropagation()} className={myModalContent}>
                {children}
```